libs/components/split-view/src/lib/modules/split-view/split-view.component.ts

```diff
@@ -75,22 +75,24 @@ export class SkySplitViewComponent implements OnInit, OnDestroy {
    */
   @Input()
   public set bindHeightToWindow(bindToHeight: boolean) {
-    this._bindHeightToWindow = bindToHeight;
+    this.#_bindHeightToWindow = bindToHeight;
+
+    if (this.#bindHeightToWindowUnsubscribe) {
+      this.#bindHeightToWindowUnsubscribe.next();
+      this.#bindHeightToWindowUnsubscribe.complete();
+    }
 
     if (bindToHeight) {
-      this.bindHeightToWindowUnsubscribe = new Subject();
-      this.adapter.bindHeightToWindow(
-        this.elementRef,
-        this.bindHeightToWindowUnsubscribe
+      this.#bindHeightToWindowUnsubscribe = new Subject();
```

Comment:
Does this `new Subject()` need a type? Or does it default to `Subject<void>`?

Reply:
It's not needed because the type is inferred from the declaration for `#bindHeightToWindowUnsubscribe`.